Demo/PQFCustomLoadersDemo/VBFRotatingSquare.m

```diff
@@ -0,0 +1,264 @@
+    [loader initialSetupWithView:view];
+    return loader;
+}
+
+- (void)showLoader
+{
+    [self performSelector:@selector(startShowingLoader) withObject:nil afterDelay:0];
+}
+
+- (void)startShowingLoader
+{
+    self.hidden = NO;
+    self.animate = YES;
+    [self generateLoader];
+    [self startAnimation];
+}
+
+- (void)hideLoader
+{
+    self.hidden = YES;
+    self.animate = NO;
+}
+
+- (void)removeLoader
+{
+    [self hideLoader];
+    [self removeFromSuperview];
+}
+
+#pragma mark - Prepare loader
+
+- (void)initialSetupWithView:(UIView *)view
+{
+    //Setting up frame
+    self.frame = view.frame;
+    self.center = CGPointMake(CGRectGetMidX(view.bounds), CGRectGetMidY(view.bounds));
+    
+    //If it is modal, background for the loader
+    if ([view isKindOfClass:[UIWindow class]]) {
+        UIView *bgView = [[UIView alloc] initWithFrame:view.bounds];
+        bgView.backgroundColor = [UIColor colorWithWhite:0.2 alpha:0.6];
+        [self addSubview:bgView];
+    }
+    
+    //Add loader to its superview
+    [view addSubview:self];
+    
+    [self.loaderView addSubview:self.label];
+    
+    //Initial Values
+    [self defaultValues];
+    
+    //Initially hidden
+    self.hidden = YES;
+}
+
+- (void)defaultValues
+{
+    [super setBackgroundColor:[UIColor clearColor]];
+    self.cornerRadius = 0;
+    self.loaderAlpha = 1.0;
+    self.loaderColor = [UIColor flatCloudsColor];
+    self.borderColor = [UIColor whiteColor];
+    self.side = 30;
+    self.duration = 0.65;
+    self.fontSize = 14.0;
+    self.rectSize = self.side * sqrt(2);
+    self.tensionValue = 1;
+}
+
+
+
+#pragma mark - Before showing
+
+- (void)generateLoader
+{
+    [self.loaderView layoutSubviews];
+    self.loaderView.frame = CGRectMake(0, 0, self.frame.size.width, self.rectSize + 30);
+    self.loaderView.center = CGPointMake(CGRectGetWidth(self.frame)/2, CGRectGetHeight(self.frame)/2);
+    self.loaderView.layer.cornerRadius  = self.cornerRadius;
+    
+    [self addSquare];
+    [self addShadow];
+    
+    if (self.label.text) [self layoutLabel];
+}
+
+- (void)layoutLabel
+{
+    self.label.textAlignment = NSTextAlignmentCenter;
+    self.label.numberOfLines = 3;
+    self.label.textColor = [UIColor whiteColor];
+    self.label.font = [UIFont systemFontOfSize:self.fontSize];
+    
+    self.loaderView.frame = CGRectMake(self.loaderView.frame.origin.x, self.loaderView.frame.origin.y, self.loaderView.frame.size.width, self.loaderView.frame.size.height + 10 + self.fontSize*2+10 );
+    self.loaderView.center = CGPointMake(CGRectGetWidth(self.frame)/2, CGRectGetHeight(self.frame)/2);
+    
+    CGFloat xPoint = CGRectGetWidth(self.loaderView.frame)/2;
+    CGFloat yPoint = CGRectGetHeight(self.loaderView.frame) - self.fontSize/2 *[self.label numberOfLines];
+    
+    self.label.frame = CGRectMake(0, 0, CGRectGetHeight(self.loaderView.frame), self.fontSize*2+10);
+    self.label.center = CGPointMake(xPoint, yPoint);
+}
+
+- (void) addSquare {
```

Comment:
could you remove the spaces after the `(void)` and move the bracket to the next line in all the methods please ? :pray: